packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1032,6 +1032,14 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 					// HACK: TinyMCE doesn't add an undo step when pasting, for unclear reasons
 					// so we manually add it here. We also can't do it immediately it seems, or
 					// else nothing is added to the stack, so do it on the next frame.
+
+					const pastedHtml = clipboard.readHTML();
+					if (pastedHtml) {
+						event.preventDefault();
+						const modifiedHtml = await processPastedHtml(pastedHtml);
+						editor.insertContent(modifiedHtml);
+					}
```

Comment:
Does that work as expected if I select some text, then press Ctrl+V? Normally it should clear the selected text and replace it with the new one.

Reply:
Yes, it does work as expected.